scripts/updateapilist.py

```diff
@@ -75,6 +82,39 @@ def __init__(self, repo: dict):
         self.release_level = repo["release_level"]
         self.distribution_name = repo["distribution_name"]
         self.issue_tracker = repo.get("issue_tracker")
+    
+    @property
+    def issue_tracker_component_id(self):
+        if not self.issue_tracker:
+            return None
+        match = re.search(r'(?:(?:\?|&)component=(\d+)|\bcomponentid:(\d+))', self.issue_tracker)
```

Comment:
You put this in one regex? OK, cool. We could have left it in two for simplicity, but either way works.

Reply:
I've split the regex into two. Thanks @vchudnov-g!